Database/Corrections/sodQuestFixes.lua

```diff
@@ -3312,6 +3312,42 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410097,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90219] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.ORC,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
+        [90220] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.TROLL,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
+        [90221] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{152608,152618,3642}},
```

Comment:
I think this is a leftover before introducing the "fake objects"?

```suggestion
            [questKeys.startedBy] = {nil,{450012}},
```